Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM ghcr.io/linuxserver/baseimage-kasmvnc:ubuntujammy
+
+# set version label
+ARG BUILD_DATE
+ARG VERSION
+ARG MULLVAD_VERSION
+LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DATE}"
+LABEL maintainer="thespad"
+
+# title
+ENV TITLE=Mullvad-Browser
+ENV MULLVAD_RELEASE_GPG_KEY="A1198702FC3E0A09A9AE5B75D5A1D4F266DE8DDF"
+ENV TORPROJECT_RELEASE_GPG_KEY="EF6E286DDA85EA2A4BA7DE684E2C6E8793298290"
+
+RUN \
+  echo "**** install packages ****" && \
+  apt-get update && \
+  apt-get install -y --no-install-recommends \
+    fontconfig \
+    gnupg \
+    ifupdown \
+    iproute2 \
+    iptables \
+    libdbus-glib-1-2 \
+    libgtk-3-0 \
+    net-tools \
+    openresolv \
+    wireguard \
+    xz-utils && \
+  echo "**** install mullvad browser ****" && \
+  mkdir -p /app && \
+  if [ -z ${MULLVAD_VERSION+x} ]; then \
+    MULLVAD_VERSION=$(curl -sX GET "https://api.github.com/repos/mullvad/mullvad-browser/releases/latest" \
+    | jq -r .name | awk -F ' ' '{print $3}'); \
+  fi && \
+  MULLVAD_URL=$(curl -sX GET "https://api.github.com/repos/mullvad/mullvad-browser/releases" \
+  | jq -r ".[] | select(.name==\"Mullvad Browser ${MULLVAD_VERSION}\") | .assets | .[] | select(.name |contains (\"linux64\")) | select(.name |contains (\"asc\")| not) | .browser_download_url") && \
+  curl -s -o \
+    /tmp/mullvad.tar.xz -L \
+    "${MULLVAD_URL}" && \
+  curl -s -o \
+    /tmp/mullvad.tar.xz.asc -L \
+    "${MULLVAD_URL}.asc" && \
+  export GNUPGHOME="$(mktemp -d)" && \
+  gpg --batch -q --recv-keys "$TORPROJECT_RELEASE_GPG_KEY" && \
+  gpg --batch -q --recv-keys "$MULLVAD_RELEASE_GPG_KEY" && \
+  if ! gpg --batch -q --verify "/tmp/mullvad.tar.xz.asc" "/tmp/mullvad.tar.xz"; then \
+    echo "File signature mismatch"; \
+    exit 1; \
+  fi && \
+  tar xf \
+    /tmp/mullvad.tar.xz -C \
+    /app --strip-components=1 && \
+  find /app -perm 700 -exec chmod 777 {} + && \
```

Comment:
I tried 644 and 755 here and it seemed to work, unless I am missing some type of test

Reply:
Good point, I can change that - I set it like that when I was doing the initial testing and forgot to revisit.